dw-date-input.js

```diff
@@ -205,10 +204,56 @@ export class DwDateInput extends DwFormElement(LitElement) {
        * Tooltip placement
        * for more see tippyJs doc: https://atomiks.github.io/tippyjs/v6/all-props/#placement
        */
-      tipPlacement: { type: String }
+      tipPlacement: { type: String },
+      
+      /**
+       * Message to show in the error color when the date input is invalid.
+       */
+      error: { type: Object },
+
+      /**
+       * Text to show the warning message.
+       */
+      warning: { type: Object },
     };
   }
 
+  /**
+   * Sets static errorMessages. Its used at application level.
+   * @param {Object} errorMessages 
+   */
+  static setErrorMessages(errorMessages) {
+    this.errorMessages = {...this.errorMessages, ...errorMessages};
+  }
+
+  get _errorMessages() {
```

Comment:
@devangdreamworld This method is not used anywhere. I think We need to pass this getter to the `date-input`.